package.json

```diff
@@ -3,18 +3,26 @@
   "version": "0.1.1",
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.28",
+    "@fortawesome/free-solid-svg-icons": "^5.13.0",
+    "@fortawesome/react-fontawesome": "^0.1.9",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-router-dom": "^5.1.2",
     "react-scripts": "3.4.1",
     "styled-components": "^5.1.0"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
+    "cy": "npm-run-all --parallel --race start cypress:open",
+    "cy:run": "npm-run-all --parallel start cypress:run",
+    "cypress:open": "cypress open",
```

Comment:
Maybe keep it symetric to the short names and call it just `cypress` or rename the short command `cy:open`?

Reply:
Good point. I now replaced npm-run-all with concurrently since npm-run-all only runs scripts while concurrrently allows to directly add CLI commands. Thus, I could reduce it to two scripts for CY:
`
"cy:open": "concurrently \"npm start\" \"cypress open\"",
    "cy:run": "concurrently \"npm start\" \"cypress run\"",
`

What do you think?

Reply:
I like it! Didn't know about concurrently before.